requirements.txt

```
@@ -43,7 +43,7 @@ python-ldap==3.1.0; sys_platform != 'win32'
 PyPDF2==1.26.0
 pyserial==3.4
 python-dateutil==2.7.3
-pytz==2019.1
+pytz==2023.3
```

```suggestion
pytz  # no version pinning  to avoid OS perturbations
```
I propose to exceptionally remove the version pinning.
That way:

-  Those who are using pip have the latest version
- Debian or Jammy users relies on the OS `tzdata` package (see [changelog](http://changelogs.ubuntu.com/changelogs/pool/main/t/tzdata/tzdata_2023c-0ubuntu0.22.04.0/changelog))

---

I like